guides/source/5_0_release_notes.md

```diff
@@ -655,11 +655,15 @@ Please refer to the [Changelog][active-job] for detailed changes.
 
 ### Notable changes
 
-*   `ActiveJob::Base.deserialize` delegates to the job class. this allows jobs
+*   `ActiveJob::Base.deserialize` delegates to the job class. This allows jobs
     to attach arbitrary metadata when they get serialized and read it back when
     they get performed.
     ([Pull Request](https://github.com/rails/rails/pull/18260))
 
+*   Add ability to configure the queue adapter on per job basis. This allows to have
```

Comment:
Needs an `on a` here as well. Don't think the second sentence adds anything to what the first one said. Except for the `affecting each other` it's just a rephrase — let's try integrating that part into the first sentence.